ci/jenkins/pipelines/prs/helpers/pr_manager/pr_checks.py

```diff
@@ -24,6 +25,49 @@ def check_pr_from_fork(self, pr_number):
 
         print(f'PR-{pr_number} is from a fork.')
 
+    def check_pr_manifest_change(self):
+        """
+        Checks if the given PR has updated any Addon and that its respective manifest
+        has been bumped.
+
+        The Regular expression will match:
+        1: + or - symbol
+        2: Addon Name
+        3: Version
+        4: Manifest
+
+        :return:
+        """
+
+        added_changes = []
+        removed_changes = []
+        p = re.compile('(-|\+)([A-Za-z]+):&AddonVersion{"(.+)",([0-9]+)},')
+
+        cmd = '''git diff --no-ext-diff --unified=0 master \
+            internal/pkg/skuba/kubernetes/versions.go \
+                | grep "AddonVersion"'''
+
+        results = subprocess.run(cmd, shell=True, 
+                    universal_newlines=True, check=True, stdout=subprocess.PIPE)
+
+        for line in results.stdout.splitlines():
+            line = line.replace('\t', '').replace(' ', '')
+            matches = p.match(line)
+            if matches[1] == "-":
+                removed_changes.append({"name": matches[2], "version": matches[3], "manifest": matches[4]})
+            else:
+                added_changes.append({"name": matches[2], "version": matches[3], "manifest": matches[4]})
+        
+        error = False
+        for r in removed_changes:
+            for a in added_changes:
+                if r["name"] == a["name"] and (r["version"] == a["version"] or  r["manifest"] == a["manifest"]):
+                    error = True
```

Comment:
you can just exit here directly

Reply:
I was expecting, to print all errors related to this before returning an error.